src/SUMMARY.md

```diff
@@ -76,8 +76,8 @@
 - [Prologue](./part-3-intro.md)
 - [Command-line arguments](./cli.md)
 - [The Rustc Driver and Interface](./rustc-driver.md)
-    - [Ex: Type checking through `rustc_interface`](./rustc-driver-interacting-with-the-ast.md)
-    - [Ex: Getting diagnostics through `rustc_interface`](./rustc-driver-getting-diagnostics.md)
+    - [Type checking through `rustc_interface`](./rustc-driver-interacting-with-the-ast.md)
+    - [Getting diagnostics through `rustc_interface`](./rustc-driver-getting-diagnostics.md)
```

Comment:
According to https://rustc-dev-guide.rust-lang.org/rustc-driver-interacting-with-the-ast.html, it means "example", could you replace them with "Example" instead?